### PR TITLE
Multi task select all in view

### PR DESCRIPTION
**Pull request by AndrewPhilbin**
Added a "select all markers in view" leaflet control to the TaskClusterMap with visibility toggled via show prop, increased height of the map in TaskBundleWidget to accommodate the new control and added some Tailwind config for new min-height. Also added html title hover tooltip.

Not sure if circled check icon is good for this or not, any input is welcome!

### CODE REVIEW

File: src/components/EnhancedMap/SelectMarkersInViewControl/SelectMarkersInViewControl.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * the control selects all task markers currently visible on the map.
+ */
+export class SelectMarkersInViewControl extends MapControl {
+  createLeafletElement(props) {
+    return new SelectMarkersInViewLeafletControl(props)
+  }
+}
+
+export default withLeaflet(injectIntl(SelectMarkersInViewControl))
+
```
Comment (jschwarz2030) [inline src/components/EnhancedMap/SelectMarkersInViewControl/SelectMarkersInViewControl.js:65]: can you remove the unnecessary spacing here?

File: src/components/EnhancedMap/SelectMarkersInViewControl/SelectMarkersInViewControl.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      <SvgSymbol
+          title={this.options.intl.formatMessage(messages.tooltip)}
+        sym="check-circled-icon"
+        className="mr-w-4 mr-h-4 mr-fill-current mr-stroke-current"
+        viewBox="0 0 20 20" 
+      />
+    </button>
+
+    )
```
Comment (jschwarz2030) [inline src/components/EnhancedMap/SelectMarkersInViewControl/SelectMarkersInViewControl.js:44]: controlContent's component indentation here looks off.

File: src/components/EnhancedMap/SelectMarkersInViewControl/SelectMarkersInViewControl.js (hunk 1 above)
Comment (AndrewPhilbin) [reply src/components/EnhancedMap/SelectMarkersInViewControl/SelectMarkersInViewControl.js:65]: Ok. The other existing control components have a space there, so I was just mimicking their format basically.

**Comment by AndrewPhilbin** [comment]
Screen Shot 2023-06-20 at 3 13 40 PM

**Pull request merged**